global_time_series/coupled_global.py

```diff
@@ -102,7 +106,7 @@ def add_trend(year, var, year1, year2, ax, format="%4.2f", lw=1, color='b',
    'vol':'{}/post/ocn/glb/ts/monthly/10yr/glb.xml'.format(case_dir),
    'name': experiment_name,
    'yoffset':0.0,
-   'yr':([21,100],),
+   'yr':([1,100],),
```

Comment:
@golaz Do we want the default `yr` to start at 1 or keep it at 20? We currently expect the user to change the `exps` variable according to what experiment they want to run. Perhaps it would be better to read in a text file of experiments the user wants to run?

Reply:
Changing to year 1 is fine. For now, the script will require manual editing.